torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1843,6 +1843,24 @@ RegisterOperators reg2({
     DEFINE_INT_OP(aten::__or__, a | b),
     DEFINE_INT_OP(aten::__xor__, a ^ b),
 
+    //all is a function that takes an iterable and returns True if all are True
+      Operator(
+        "aten::all(Tensor x) -> bool",
```

Comment:
There's already an operator for `aten::all` that's generated from an ATen operator (`tensor.all()`), you can find it in `register_generated_aten_ops_1.cpp` after you build the code. For that reason I don't think we need any code here for this op, just adding it to the list of globals in `compiler.cpp` should be enough.

Reply:
Sorry I'm such a noob but hopefully this looks ok. If not can you give me a little bit more detail on the whitespace cleanup. Thanks